eth_tester/backends/mock/serializers.py

```diff
@@ -52,6 +52,13 @@ def serialize_full_transaction(transaction, block, transaction_index, is_pending
         partial(assoc, key="type", value=extract_transaction_type(transaction)),
     )
 
+    if int(serialized_transaction["type"], 16) > 0:
+        serialized_transaction = assoc(
+            serialized_transaction,
```

Comment:
This was a bonus that I noticed providers return. There is still a `v` field for non-legacy transactions but there's also the correct `y_parity` field of the signature.